app/javascript/packs/user.js

```diff
@@ -66,6 +66,24 @@ $(document).ready(function(){
     }
   });
 
+  if($('input[name="add_address_other"]').is(':checked'))
+  {
+    $('.other-address').show();
+  }else
+  {
+    $('.other-address').hide();
+  }
+
+  $('input[name="add_address_other"').change(function(){
+    if($('input[name="add_address_other"]').is(':checked'))
```

Comment:
- dùng thử toggle đc ko nha
- đang show thì sẽ hide và ngược lại
```suggestion
    if($('input[name="add_address_other"]').is(':checked'))
    {
      $('.other-address').toggle();
    }
```